server/tserver/src/main/java/org/apache/accumulo/tserver/TabletClientHandler.java

```diff
@@ -1214,16 +1214,14 @@ public void halt(TInfo tinfo, TCredentials credentials, String lock)
 
     checkPermission(security, context, server, credentials, lock, "halt");
 
-    Halt.halt(0, () -> {
```

Comment:
This is called from `fastHalt` below and removing the call to `Halt` here would be a behavior change as it would not stop the tserver immediately. Not sure why we have two methods that do the same thing, likely they were different at some point.